### PR TITLE
doc: change suggested scripts names

### PR DESCRIPTION
**Pull request by GantMan**
The script name suggested (`add`) will conflict with users of yarn.

So I modified the suggested name to `append` and provided a yarn example as well.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,20 +19,30 @@ Once initialized, you don't need to have  `all-contributors-cli` installed globa
...
+yarn add all-contributors-cli --dev
+```
+
 ```json
 {
   "scripts": {
-    "add": "all-contributors add",
+    "append": "all-contributors add",
     "generate": "all-contributors generate"
   }
```
Comment (machour) [inline README.md]: Really good point.
Let's add `contributors:` as prefix instead tho:

```
    "contributors:add": "all-contributors add",
    "contributors:generate": "all-contributors generate",
```

File: README.md [hunk 2]
```diff
@@ -19,20 +19,30 @@ Once initialized, you don't need to have  `all-contributors-cli` installed globa
...
 }
 ```
 and use them via `npm run`:
 ```console
-npm run add -- jfmengels doc
+npm run append -- jfmengels doc
 npm run generate
 ```
 
+OR use them via `yarn`:
```
Comment (machour) [inline README.md]: really not sure we want to duplicate all examples (ref: my first comment)

File: README.md [hunk 3]
```diff
@@ -19,20 +19,30 @@ Once initialized, you don't need to have  `all-contributors-cli` installed globa
 ```console
 npm install --save-dev all-contributors-cli
 ```
+```console
```
Comment (machour) [inline README.md]: I'm not sure about adding yarn examples, as it make the documentation bloated in my opinion.
People who use yarn know how to convert a npm command, don't they? 🤔

File: README.md [hunk 4]
```diff
@@ -19,20 +19,30 @@ Once initialized, you don't need to have  `all-contributors-cli` installed globa
...
+    "append": "all-contributors add",
     "generate": "all-contributors generate"
   }
 }
 ```
 and use them via `npm run`:
 ```console
-npm run add -- jfmengels doc
+npm run append -- jfmengels doc
 npm run generate
```
Comment (machour) [inline README.md]: don't forget to update these with the new scripts names

**Comment by GantMan** [comment]
@machour - should be 💯

**Pull request merged**